chat/models.py

```diff
@@ -21,3 +21,11 @@ class Message(models.Model):
     deleted = models.BooleanField(default=False)
     tokens = models.IntegerField(default=0)
     created_at = models.DateTimeField(auto_now_add=True)
+
+
+class SQLQuery(models.Model):
```

Comment:
I would also create a model for the DB connection. 
That way a user can configure a new connection from the UI, set up the urls and create a schema.
Then you can save the schema in the db and use it when making the query